src/Consolonia.Themes/Templates/Controls/AutoCompleteBox.axaml

```diff
@@ -1,14 +1,20 @@
 <ResourceDictionary xmlns="https://github.com/avaloniaui"
-                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:console="https://github.com/jinek/consolonia"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                     x:ClassModifier="internal">
     <ControlTheme x:Key="{x:Type AutoCompleteBox}"
                   TargetType="AutoCompleteBox">
         <Setter Property="Background"
                 Value="{DynamicResource ThemeSelectionBackgroundBrush}" />
         <Setter Property="Foreground"
                 Value="{DynamicResource ThemeSelectionForegroundBrush}" />
-        <Setter Property="BorderBrush"
-                Value="{DynamicResource ThemeChooserBackgroundBrush}" />
+        <Setter Property="BorderBrush" >
```

Comment:
It seems AutoCompleteBox does not have border ever, why do we set here it to be 1 and custom color?

Reply:
you are right, removed